src/dmd/dinifile.d

```diff
@@ -229,10 +229,10 @@ void parseConfFile(StringTable* environment, const(char)* filename, const(char)*
         for (size_t k = 0; k < i - linestart; ++k)
         {
             // The line is buffer[linestart..i]
-            char* line = cast(char*)&buffer[linestart];
+            const(char)* line = cast(char*)&buffer[linestart];
```

Comment:
Suggest:
```D
const line = cast(const char*)&buffer[linestart];
```
If you need to reassign `line`, use `auto` instead of the leftmost `const`.